### PR TITLE
Preprocessing

### PR DESCRIPTION
**Pull request by wronaq**

### CODE REVIEW

File: ai/README.md [hunk 1]
```diff
@@ -16,11 +16,20 @@ pdm sync
...
+
+## Abstract cleaner
+
+To preprocess and clean abstracts downloaded from [arxiv.org](https://arxiv.org/) one should run the scrip with a code from below:
+
+```bash
+make clean-abstracts path=<PATH-TO-SCRAPED-ARXIV-DATASET>
+```
+
+Output will be a pain text file (abstract per row) that will be saved to `ai/data` directory with `_cleaned.txt` suffix, log will be stored in `ai/logs`.
```
Comment (MichalBortkiewicz) [inline ai/README.md]: pain -> plain

**Review by MichalBortkiewicz — COMMENTED** [review]
if I understand, correctly you got rid of latex and stuff in the preprocessed dataset? It could be also nice to show it in the notebook. 
Anyway I admire the good coding practices, it is super refreshing after working only with 'scientific' code for a while ;P

File: ai/README.md (hunk 1 above)
Comment (wronaq) [reply ai/README.md]: Fixed

**Comment by wronaq** [comment]
> if I understand, correctly you got rid of latex and stuff in the preprocessed dataset? It could be also nice to show it in the notebook

Yes, exactly. I've added a jupyter notebook called `simple_eda.ipynb` into the `notebooks` diecrectory. I will also add comment to the `README.md`.

**Pull request merged**